lib/presentation/widget/received/received_share_widget.dart

```diff
@@ -242,11 +243,38 @@ class _ReceivedShareWidgetState extends State<ReceivedShareWidget> {
           SvgPicture.asset(receivedShareItem.element.mediaType.getFileTypeImagePath(imagePath),
               width: 20, height: 24, fit: BoxFit.fill)
         ]),
-        title: Transform(
-          transform: Matrix4.translationValues(-16, 0.0, 0.0),
-          child: _buildFileName(receivedShareItem.element.name),
+        title: ResponsiveWidget(
+          smallScreen: Transform(
+            transform: Matrix4.translationValues(-16, 0.0, 0.0),
+            child: _buildFileName(receivedShareItem.element.name),
+          ),
+          mediumScreen: Transform(
+              transform: Matrix4.translationValues(-16, 0.0, 0.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Expanded(
+                    child: Transform(
+                      transform: Matrix4.translationValues(0.0, -14.0, 0.0),
+                      child:  _buildFileName(receivedShareItem.element.name),
+                    ),
+                    flex: 2,
+                  ),
+                  Expanded(
+                    child: _buildSenderName(receivedShareItem.element.sender.fullName()),
+                    flex: 1,
+                  ),
+                  Expanded(
+                    child: _buildModifiedDateText(AppLocalizations.of(context).item_created_date(
+                        receivedShareItem.element.creationDate.getMMMddyyyyFormatString())),
+                    flex: 1,
+                  )
+                ],
+              )
+          ),
         ),
-        subtitle: Transform(
+        subtitle: ResponsiveWidget.isSmallScreen(context)
+            ? Transform(
           transform: Matrix4.translationValues(-16, 0.0, 0.0),
```

Comment:
pls fix indention

Reply:
I fixed it and push commit fixup. Thank you, please